project/eScavenge/models.py

```diff
@@ -164,14 +164,17 @@ def set_time_penalty(self, penalty):
         return True
 
     def start(self):
-        now = timezone.now()
-        for team in self.teams.all():
-            team.clue_time = now
-            team.full_clean()
-            team.save()
-        self.started = True
-        self.full_clean()
-        self.save()
+        if not self.started:
```

Comment:
Fail fast. If game is already started we can return immediately, don’t need to add an extra indent block for everything else.